src/main/java/levy/gameoflife/Main.java

```diff
@@ -0,0 +1,6 @@
+package levy.gameoflife;
+
+public class Main {   public static void main(String[] args) { new GameOfLifeFrame().setVisible(true);
+}
+}
```

Comment:
Bad formatting.

Feels like you copied it from somewhere., you obviously didn't write it like this.

Reply:
I copied from my sand project

Reply:
thats how i have it in there but i will fix